src/test/javascript/spec/component/tutorial-groups/tutorial-groups-management.component.spec.ts

```diff
@@ -0,0 +1,114 @@
+    @Input() tutorialGroup: TutorialGroup;
+    @Input() courseId: number;
+}
+describe('TutorialGroupsManagementComponent', () => {
+    let tutorialGroupsManagementComponentFixture: ComponentFixture<TutorialGroupsManagementComponent>;
+    let tutorialGroupsManagementComponent: TutorialGroupsManagementComponent;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [
+                TutorialGroupsManagementComponent,
+                LoadingIndicatorContainerStubComponent,
+                TutorialGroupRowButtonsStubComponent,
+                MockPipe(ArtemisTranslatePipe),
+                MockComponent(FaIconComponent),
+                MockRouterLinkDirective,
+                MockDirective(SortDirective),
+                MockDirective(SortByDirective),
+            ],
+            providers: [
+                MockProvider(TutorialGroupsService),
+                MockProvider(AlertService),
+                MockProvider(SortService),
+                { provide: Router, useClass: MockRouter },
+                {
+                    provide: ActivatedRoute,
+                    useValue: {
+                        parent: {
+                            paramMap: of({
+                                get: (key: string) => {
+                                    switch (key) {
+                                        case 'courseId':
+                                            return 1;
+                                    }
+                                },
+                            }),
+                        },
+                    },
+                },
+            ],
+        })
+            .compileComponents()
+            .then(() => {
+                tutorialGroupsManagementComponentFixture = TestBed.createComponent(TutorialGroupsManagementComponent);
+                tutorialGroupsManagementComponent = tutorialGroupsManagementComponentFixture.componentInstance;
+            });
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should initialize', () => {
+        tutorialGroupsManagementComponentFixture.detectChanges();
+        expect(tutorialGroupsManagementComponent).not.toBeNull();
+    });
+
+    it('should get all tutorial groups for course', () => {
+        const exampleTutorialGroup = new TutorialGroup();
+        exampleTutorialGroup.id = 1;
+
+        const tutorialGroupsService = TestBed.inject(TutorialGroupsService);
+        const getAllForCourseSpy = jest.spyOn(tutorialGroupsService, 'getAllOfCourse').mockReturnValue(
+            of(
+                new HttpResponse({
+                    body: [exampleTutorialGroup],
+                    status: 200,
+                }),
+            ),
+        );
+
+        tutorialGroupsManagementComponentFixture.detectChanges();
+        expect(tutorialGroupsManagementComponent.tutorialGroups).toEqual([exampleTutorialGroup]);
+        expect(getAllForCourseSpy).toHaveBeenCalledWith(1);
```

Comment:
Missing `toHaveBeenCalledOnce()`

Reply:
Done in d9aae13fd3c1aa34fdf4330d8e95e9c445163825